tests/python/contrib/test_hexagon/test_meta_schedule.py

```diff
@@ -303,23 +309,20 @@ def test_vrmpy_dense_auto_tensorize(hexagon_launcher):
 
     if True:
         with tempfile.TemporaryDirectory() as work_dir:
-            config = ms.TuneConfig(
-                strategy="replay_trace",
+            target = get_hexagon_target("v68")
+            database = ms.tir_integration.tune_tir(
```

Comment:
Two uses of `tune_tir` in this file have incorrect signature. I got the following errors:
```
E               TypeError: tune_tir() got an unexpected keyword argument 'sch_rules' 
```

```
E             Check failed: (!checked_type.defined()) is false: Expected Map[meta_schedule.Mutator, FloatImm], but got Array
```

Reply:
this should work:

```
            target = get_hexagon_target("v68")
            database = ms.tir_integration.tune_tir(
                mod=workload,
                target=target,
                max_trials_global=8,
                num_trials_per_iter=8,
                max_trials_per_task=8,
                work_dir=work_dir,
                space=ms.space_generator.PostOrderApply(
                    f_block_filter=None,
                    sch_rules=sch_rules,
                    postprocs=postprocs,
                    mutator_probs={},
                ),
                builder=get_hexagon_local_builder(),
                runner=get_hexagon_rpc_runner(hexagon_launcher, number=10),
            )
            sch = ms.tir_integration.compile_tir(database, workload, target)

```